web/src/components/navbar/NavBar.tsx

```diff
@@ -37,8 +37,21 @@ export const NavBar = (props: Props): ReactElement => {
 
   if (props.logoOnly) {
     return <NavBarLogoOnly />;
-  } else if (props.landingPage) {
+  } else if (props.landingPage && isLargeScreen) {
     return <NavBarLanding />;
+  } else if (props.landingPage && !isLargeScreen) {
```

Comment:
This becomes a little confusing naming-wise because previously `NavBarLanding` was on the landing page. 

Now, if we're mobile, `NavBarLanding` is NOT rendered, `NavBarMobile` is.

Maybe we should call it `NavBarLandingDesktop` or something?